images/sdn/Dockerfile.rhel

```diff
@@ -25,4 +27,4 @@ RUN INSTALL_PKGS=" \
 LABEL io.k8s.display-name="OpenShift SDN" \
       io.k8s.description="This is a component of OpenShift and contains the networking tool stack for the default SDN implementation." \
       io.openshift.tags="openshift,sdn"
-ENTRYPOINT [ "/usr/local/bin/openshift-sdn" ]
+ENTRYPOINT [ "/usr/bin/openshift-sdn" ]
```

Comment:
Was this change intentional? Doesn't seem related to other changes.

Reply:
It was intentional, if you look further up in the dockerfile you can see that openshift-sdn was put in usr/bin and /usr/local/bin. Would it be better to separate that into its own commit?

Reply:
Sure, go ahead and separate it into a unique commit so there's some context around that specific change. Is the ENTRYPOINT used or overridden elsewhere? I'm confused as to how it would've been working prior to your change.

Reply:
RHEL 7 has unified /usr/bin and /bin so functionally they are the same but it is good practice to refer to the binary as in the same location all the time. I submitted a PR to the operator to do the same.

https://github.com/openshift/cluster-network-operator/pull/128